lib/gmp/native_gmp.nit

```diff
@@ -0,0 +1,266 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Low-level GMP features
+module native_gmp is ldflags("-lgmp")
+
+in "C header" `{
+    #include <gmp.h>
+`}
+
+# Multy precision Integer
+extern class NativeLong `{mpz_ptr`}
```

Comment:
Why not keep the native name? either `MPZ` or `NativeMPZ`. (See the next comment, and you don't need the `Native` part if you change the name in the Nity layer.)

And there's a typo in *multi.